robber/matchers/__init__.py

```diff
@@ -5,12 +5,12 @@
     'called_once',
     'called_once_with',
     'called_with',
-    'contain'
+    'contain',
     'equal',
-    'ever_called'
+    'ever_called',
```

Comment:
I thought of it but it just sounds weird and awkward. So I think, it's better to keep `ever_called_with` and add `have.any_call` as an equivalent chain.